Makefile

```diff
@@ -212,7 +212,7 @@ clean_meta:
 #   make clean_generated
 .PHONY: clean_generated
 clean_generated:
-	find . -type f -name $(GENERATED_FILE_PREFIX)\* | xargs rm -f
+	find . -type f -name $(GENERATED_FILE_PREFIX)\* | grep -v "[.]/staging/.*" | xargs rm -f
```

Comment:
because generate does not generate them. Adding a comment can be useful, but the unwritten logic for clean_generated and generated_files is, whatever generated_files generates, clean_generated should clean.